apps/myaccount/src/components/federated-associations/federated-associations.tsx

```diff
@@ -103,8 +103,10 @@ export const FederatedAssociations: FunctionComponent<FederatedAssociationsProps
      */
     useEffect(() => {
         if (disableExternalLoginsOnEmpty) {
-            if (federatedAssociations) {
+            if (federatedAssociations?.length <= 0) {
```

Comment:
Why do we need a length check here?

Reply:
The condition is based on whether we have any **federatedAssociations** in the list.